README.md

```diff
@@ -30,3 +30,5 @@ If you're curious about the tools, stacks and technologies we use, get ready for
 - apply directly via [our hiring page](http://carrot.is/hiring)
 - [file an issue](https://github.com/carrot/we-are-hiring/issues) with a question you might have
 - fork this repository and PR in your resume/application (or just something we might find interesting)
+
+Last Updated: 04-10-2015 08:27 AM
```

Comment:
don't we already know when it was last updated from the git commit? Like `git log -1 --format=%cd README.md`?